.golangci.yml

```diff
@@ -36,7 +35,7 @@ linters:
   disable-all: true
   enable:
     - gofmt         # Checks whether code was gofmt-ed
-    - golint        # Differs from gofmt. Gofmt reformats Go source code, whereas golint prints out style mistakes
+    # - revive        # Differs from gofmt. Gofmt reformats Go source code, whereas golint prints out style mistakes
```

Comment:
Why is this commented out?

Reply:
Oh, that left from my attempt from testing revive and received plethora of errors. I can revert it, but `golint` also was not working anyway. 

```
WARN [linters_context] golint: This linter is fully inactivated: it will not produce any reports.
```

I can spend some time to set useful checks from revive and fix the complaints. Probably in a separate PR.

Reply:
That's fine but we should have a ticket if we defer it so we don't forget about it.